package.json

```diff
@@ -14,6 +14,8 @@
   },
   "dependencies": {
     "buffer": "^6.0.3",
+    "delay": "^5.0.0",
```

Comment:
maybe we could use promise and save on the module?

```js
await new Promise(r => setTImeout(r, 1000))
```

Reply:
We've used `delay` but sure we can remove